src/services/connection/connection.service.ts

```diff
@@ -0,0 +1,56 @@
+import { dataPoolApi } from "../../api/data-pool-api";
+import { logger } from "../../util/logger";
+
+class ConnectionService {
+    public async findAllConnections(dataPoolId: string): Promise<any[]> {
+        const connections =  await dataPoolApi.listConnections(dataPoolId);
+        connections.forEach(connection => {
+            logger.info(`Connection ID: ${connection.id} - Name: ${connection.name} - Type: ${connection.type}`);
```

Comment:
What's a typical number of connections returned here?
Wondering if this could clutter the logs and make it unreadable.